### PR TITLE
Initialize README.md

### PR DESCRIPTION
**Pull request by juliaogris**
Initialize README.md with basic information on auto-generated repo content
and links for background reading on homebrew.

Add Apache 2.0 License

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Homebrew formulas
+
+This repository contains automatically generated [homebrew]
+[formulas] for various Foxygoat tools and CLIs.
+
+[homebrew]: https://brew.sh
```
Comment (camh-) [inline README.md]: I like to put the link anchors at the end of the section (before the next heading). Up to you though.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+[formulas]: https://docs.brew.sh/Formula-Cookbook
+
+The formulas can be used with 
+
+	brew tap foxygoat/tap
```
Comment (camh-) [inline README.md]: These are indented with tab, not 4 spaces. I can't remember what, if anything, we decided about that, but I know I always use 4 spaces. The markdown guide says "indenting lines by four spaces or one tab" so either is fine.

**Review by camh- — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (juliaogris) [reply README.md]: fixed.

File: README.md (hunk 2 above)
Comment (juliaogris) [reply README.md]: fixed.

**Pull request merged**